kindlecomicconverter/KCC_gui.py

```diff
@@ -242,6 +242,7 @@ def run(self):
         options.cropping = GUI.croppingBox.checkState().value
         if GUI.croppingBox.checkState() != Qt.CheckState.Unchecked:
             options.croppingp = float(GUI.croppingPowerValue)
+        options.interpanelcrop = GUI.interPanelCropBox.checkState().value
```

Comment:
- ref #785 

You aren't saving the value on close.

Maybe this PR should be added as example of adding new checkbox in the readme?